### PR TITLE
Tictec 2020

### PR DESCRIPTION
**Pull request by wrightmartin**

### CODE REVIEW

File: ainow/templates/ainow/_nav.html [hunk 1]
```diff
@@ -50,4 +53,11 @@
 <li><a href="{% url 'sponsorship' %}">Sponsorship</a></li>
 <li><a href="{% url 'press' %}">Press Enquiries</a></li>
 
+{% elif schedule.slug == '2020' %}
+
+<li><a href="{% url 'schedule2020summary' %}">Summary</a></li>
+<li><a href="{% url 'sponsorship' %}">Sponsorship</a></li>
+<li><a href="{% url 'press' %}">Press Enquiries</a></li>
+<li><a href="https://www.mysociety.org/tag/tictec/">Blog</a></li>
+
```
Comment (chrismytton) [inline ainow/templates/ainow/_nav.html:62]: The top nav looks a bit broken when the screen is narrower than about 990px but before the mobile breakpoint kicks in:

image

**Comment by chrismytton** [comment]
Just a note to myself: I had to create a `Schedule` with the slug `2020` to get this working properly, so that will need to be done on the production server **before** this is deployed.

File: ainow/templates/ainow/2020-summary.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
+{% extends 'ainow/base.html' %}
+{% load staticfiles %}
+
+{% block headerextra %}
+<meta property="og:title" content="TICTeC 2020" />
+<meta property="og:type" content="website" />
+<meta property="og:url" content="https://tictec.mysociety.org/2020" />
+<meta property="og:image" content="https://tictec.mysociety.org/static/img/local-og.jpg" />
```
Comment (chrismytton) [inline ainow/templates/ainow/2020-summary.html]: This image seems to be about TICTeC local, does this need updating?

File: ainow/templates/ainow/2020-summary.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+</div>
+
+<div id="intro" class="homepage-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-8">
+
+        <p class="lead">Returning for a sixth year, TICTeC 2020 will focus on the impact that civic technology and digital democracy are having upon citizens, decision makers and governments around the world. </p>
+
+        <p>Primarily, the goal of TICTeC is to promote and share rigorous and meaningful research into online civic technologies and digital democracy around the world. The conference facilitates discussion and networking amongst individuals and groups to find real-world solutions through sharing evidence of impact, and (importantly) evidence of what doesn’t work. Read how attendees have benefited from coming to TICTeC over the years. </p>
```
Comment (chrismytton) [inline ainow/templates/ainow/2020-summary.html]: > Read how attendees have benefited from coming to TICTeC over the years

Is this supposed to link to something?

File: ainow/templates/ainow/2020-summary.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  </div>
+</div>
+
+<div id="sponsorship" class="homepage-section homepage-section--secondary">
+    <div class="container">
+      <h2>Sponsorship opportunities</h2>
+      <div class="row">
+        <div class="col-sm-8">
+  
+          <p>mySociety is a charity and relies on sponsorship and ticket sales to make events like TICTeC happen. If you’re interested in sponsoring TICTeC 2020 in Reykjavik please see our <a href="http://tictec.mysociety.org/sponsorship#attendees">sponsorship options</a>. To discuss these and other more bespoke sponsorship opportunities please contact <a href="mailto:gemma@mysociety.org">contact Gemma Moulder</a>.
```
Comment (chrismytton) [inline ainow/templates/ainow/2020-summary.html]: > …sponsorship opportunities please **contact contact** Gemma Moulder.

Duplication of the word contact here.

**Review by chrismytton — CHANGES_REQUESTED** [review]
Looking good @wrightmartin! Just a few hopefully minor suggestions I've made.

**Comment by chrismytton** [comment]
> Just a note to myself: I had to create a `Schedule` with the slug `2020` to get this working properly, so that will need to be done on the production server **before** this is deployed.

I've done this on the live site, so we should be good to just deploy this.

Note that when this change gets deployed it will show up on the live site immediately. Not sure if that's what we want, or whether we want to do a big reveal when we start promoting it, @Gemmamysoc?

**Comment by Gemmamysoc** [comment]
cheers! Happy for it to go live from now if it's ready @chrismytton !

File: ainow/templates/ainow/_nav.html (hunk 1 above)
Comment (wrightmartin) [reply ainow/templates/ainow/_nav.html:62]: This is an annoying one, mainly due to the number of pages we're linking to - I wonder how we'd feel about just ignoring this for now, as once more details are finalised and the date moves closer the navigation will change again

Comment (wrightmartin) [reply ainow/templates/ainow/_nav.html:62]: Rather than adding CSS for this - seems edge-casey

**Comment by chrismytton** [comment]
This is now live at https://tictec.mysociety.org/2020 :tada:

**Pull request merged**